src/Resources/public/css/main.css

```diff
@@ -163,8 +163,10 @@
     margin: 16px 0 10px 0;
 }
 
-
-
 #guzzle_profiler .content {
     padding: 10px;
 }
+
+.content.hide {
```

Comment:
why not adding #guzzle_profiler in front? just to avoid conflicts with other css styles...

Reply:
yep, I thought about that and it's next that I planned to do :slightly_smiling_face:

Reply:
Now the main question is about js part